### DISCUSSION TITLE
<title> Changing Logo

### DISCUSSION THREAD

**Discussion by eadnimri** [community/community#45255] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I am a mosaic artist and i have a GitHub business page and i was wondering how can i change my logo icon

**Comment by SilasRodrigues19** [comment]
Hey @eadnimri 

Try this:

`<link rel="shortcut icon" href="favicon.ico" type="image/x-icon">`

Just change the file path

**Comment by eadnimri** [comment]
I need help with that bc i am still new to GitHub Ead’s iPhoneOn Jan 24, 2023, at 1:06 PM, Silas Rodrigues ***@***.***> wrote:﻿
Hey @eadnimri
Try this:
<link rel="shortcut icon" href="favicon.ico" type="image/x-icon">
Just change the file path

**Reply by samar19947** [reply]
😞

**Comment by eadnimri** [comment]
I was going can i create a QR Code on GitHub Ead’s iPhoneOn Jan 24, 2023, at 1:26 PM, samar19947 ***@***.***> wrote:﻿
😞
